Src/Screens/ProfileScreen/Screens/CharityScreen/Viewables/CharityInformationBody.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  CharityInformationBody.swift
+//  ugglan
+//
+//  Created by Gustaf Gunér on 2019-04-04.
+//
+
+import Foundation
+import Flow
+import UIKit
+import MarkdownKit
+
+struct CharityInformationBody {
+    let text: String
```

Comment:
I've been kinda leaning against always taking in `Signals`, up to you but it makes the components a bit more flexible.